migration/v0.24.x-v0.25.x/add-control-plane-toleration.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -euo pipefail
+
+: "${CK8S_CONFIG_PATH:?Missing CK8S_CONFIG_PATH}"
+
+common_config="${CK8S_CONFIG_PATH}/common-config.yaml"
+wc_config="${CK8S_CONFIG_PATH}/wc-config.yaml"
+
+add_toleration() {
+    yq4 -i '((.. | select(key == "tolerations") | (select(tag == "!!seq" and contains([{"key": "'"$1"'"}]) and contains([{"key": "'"$2"'"}]) == false))) += [{"key": "'"$2"'", "operator": "Exists", "effect": "NoSchedule"}])' "$3"
+}
+
+for i in ${wc_config} ${common_config}
```

Comment:
Maybe add sc-config to the loop as well? just in case there is some cluster that has added master toleration there.